build.sbt

```diff
@@ -34,15 +65,15 @@ lazy val loader = project
   .settings(addCompilerPlugin(Dependencies.betterMonadicFor))
   .settings(libraryDependencies ++= Dependencies.loaderDependencies)
   .settings(excludeDependencies ++= Dependencies.exclusions)
-  .dependsOn(common % "compile->compile;test->test", aws)
+  .dependsOn(common % "compile->compile;test->test", aws % "compile->compile;test->test", gcp % "compile->compile;test->test")
   .enablePlugins(JavaAppPackaging, DockerPlugin, BuildInfoPlugin)
 
 lazy val redshiftLoader = project
   .in(file("modules/redshift-loader"))
   .settings(BuildSettings.redshiftBuildSettings)
   .settings(addCompilerPlugin(Dependencies.betterMonadicFor))
   .settings(libraryDependencies ++= Dependencies.redshiftDependencies)
-  .dependsOn(common % "compile->compile;test->test", aws, loader % "compile->compile;test->test")
+  .dependsOn(common % "compile->compile;test->test", loader % "compile->compile;test->test")
```

Comment:
`common % "compile->compile;test->test",` can be removed here and in `redshiftLoaderDistroless` as well